### DISCUSSION TITLE
ESM imports not working

### DISCUSSION THREAD

**Discussion by LucasC-S** [parcel-bundler/parcel#9462] [Q&A] [answered]
Not sure if this is 100% related to parcel or the packages themselves, but I am trying to use remark-math and rehype-katex with a react app I am using parcel to build. However, when I try and build I get a number of errors where the modules themselves have failed imports. Here is an example:

```
🚨 Build failed.

@parcel/core: Failed to resolve 'devlop' from './node_modules/hast-util-from-html/lib/index.js'

./node_modules/hast-util-from-html/lib/index.js:60:28
    59 | 
  > 60 | import {ok as assert} from 'devlop'
  >    |                            ^^^^^^^^
    61 | import {fromParse5} from 'hast-util-from-parse5'
    62 | import {parse, parseFragment} from 'parse5'

@parcel/resolver-default: Cannot load file './index' from module 'devlop'
```

The stuff I have found about this from the unified team all indicates that it is a problem with the resolver (eg. this also happens with jest) and not any unified packages. Any help would be greatly appreciated!!

**Comment by mischnic** [comment, marked as answer]
devlop uses package exports: https://unpkg.com/browse/devlop@1.1.0/package.json

You have to opt into them when using Parcel: https://parceljs.org/features/dependency-resolution/#package-exports
